voltixApp/VoltixApp/Utils/Utils.swift

```diff
@@ -94,99 +94,179 @@ enum Utils {
                 completion(.failure(error))
             }
 			
-            guard let httpResponse = response as? HTTPURLResponse else {
-                completion(.failure(NSError(domain: "Invalid response", code: 0, userInfo: nil)))
-                return
-            }
-            switch httpResponse.statusCode {
-                case 200 ... 299:
-                    guard let data = data else {
-                        completion(.failure(NSError(domain: "No data available", code: 0, userInfo: nil)))
-                        return
-                    }
-                    completion(.success(data))
-                case 404: // success
-                    completion(.failure(NSError(domain: "Invalid response code", code: httpResponse.statusCode, userInfo: nil)))
-                    return
-                default:
-                    completion(.failure(NSError(domain: "Invalid response code", code: httpResponse.statusCode, userInfo: nil)))
-                    return
-            }
+			guard let httpResponse = response as? HTTPURLResponse else {
+				completion(.failure(NSError(domain: "Invalid response", code: 0, userInfo: nil)))
+				return
+			}
+			switch httpResponse.statusCode {
+				case 200 ... 299:
+					guard let data = data else {
+						completion(.failure(NSError(domain: "No data available", code: 0, userInfo: nil)))
+						return
+					}
+					completion(.success(data))
+				case 404: // success
+					completion(.failure(NSError(domain: "Invalid response code", code: httpResponse.statusCode, userInfo: nil)))
+					return
+				default:
+					completion(.failure(NSError(domain: "Invalid response code", code: httpResponse.statusCode, userInfo: nil)))
+					return
+			}
 			
-        }.resume()
-    }
+		}.resume()
+	}
 	
-    public static func getMessageBodyHash(msg: String) -> String {
-        let digest = Insecure.MD5.hash(data: Data(msg.utf8))
-        return digest.map {
-            String(format: "%02hhx", $0)
-        }.joined()
-    }
+	static func fetchArray<T: Decodable>(from urlString: String) async throws -> [T] {
+		do {
+			let data = try await Utils.asyncGetRequest(urlString: urlString, headers: [:])
+			print("fetchArray")
```

Comment:
Can you please remove this debug statement?

Reply:
Done